trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -69,6 +69,13 @@ public class GravitinoConfig {
           "true",
           false);
 
+  private static final ConfigEntry GRAVITINO_CLUSTER =
+      new ConfigEntry(
+          "gravitino.cluster",
+          "The cluster name of the trino, if it's empty, all catalogs will be handled locally",
```

Comment:
It seems strange that `gravitino.cluster` is referred to as `The cluster name of the trino`.
BTW, Where should the user obtain the cluster name?

Reply:
This will need to be considered by our Gravitino enterprise product in the future. For now, let's use the cluster name with the catalog's properties

Reply:
Can you provide an example here?

Reply:
We can create a catalog with a cluster information:
```
call gravitino.system.create_catalog(
    'gt_mysql',
    'jdbc-mysql',
    map(
        array['jdbc-url', 'jdbc-user', 'jdbc-password', 'jdbc-driver', 'cluster', 'cluster.connection-url', 'cluster.connection-user'],
        array['jdbc:mysql://mysql/?useSSL=false', 'trino', 'ds123', 'com.mysql.cj.jdbc.Driver', 'c2', 'jdbc:trino://trino-remote:8080', 'admin']
    )
);
```

When the gravitino-trino-connector config like:
```
#Thu Jun 13 17:50:55 CST 2024
connector.name=gravitino
gravitino.metalake=test
gravitino.uri=http\://gt01.orb.local\:8090
trino.catalog.store=etc/catalog
trino.jdbc.uri=jdbc\:trino\://localhost\:8080
gravitino.cluster=c2
```

If the catalog's cluster name equals then the gravition-trino-connector 's  cluster configuration. The catalog is handle by local